test/karma/test-app/es5-addclass-svg/karma.spec.ts

```diff
@@ -1,31 +0,0 @@
-import { setupDomTests } from '../util';
-
-describe('es5 $addClass svg', () => {
-  const { setupDom, tearDownDom } = setupDomTests(document);
-  let element: HTMLElement;
-
-  beforeEach(async () => {
-    const app = await setupDom('/es5-addclass-svg/index.html');
-    element = app.querySelector('es5-addclass-svg');
-  });
-  afterEach(tearDownDom);
-
-  it('should add a class', async () => {
-    if ('attachShadow' in HTMLElement.prototype) {
-      testNativeShadow(element);
-    } else {
-      testPolyfilledShadow(element);
-    }
-  });
-
-  function testNativeShadow(result: HTMLElement) {
-    const svg = result.shadowRoot.querySelector('svg');
-    expect(svg.getAttribute('class')).toContain('existing-css-class');
-    expect(svg.getAttribute('class')).not.toContain('sc-es5-addclass-svg');
-  }
-
-  function testPolyfilledShadow(result: HTMLElement) {
```

Comment:
Can we add a note to the git commit as to why we're no longer testing this please?

Reply:
Rebased the branch and squashed all commits into a single one that contains a note that we removed testing this behavior as we expect Shadow DOM to be widely supported by now.